libs/pages/applications/ui/src/lib/components/container/container.tsx

```diff
@@ -9,19 +9,21 @@ import {
   Header,
   Icon,
   StatusMenu,
-  StatusMenuState,
   Tabs,
   Tag,
+  StatusChip,
+  Skeleton,
 } from '@console/shared/ui'
 import { IconEnum } from '@console/shared/enums'
 import { ClickEvent } from '@szhsin/react-menu'
 
 export interface ContainerProps {
   applications: Application[]
+  environment?: Environment & { status?: Status }
```

Comment:
Example why we should have EnvironmentEntity because you don't want to come back in all the containerProps to add the new objects we add on the entity. We need to factorize this.